packages/ember-glimmer/tests/utils/test-helpers.js

```diff
@@ -83,7 +83,7 @@ export function equalsElement(element, tagName, attributes, content) {
     QUnit.push(element instanceof HTMLElement, null, null, 'Element must be an HTML Element, not an SVG Element');
   } else {
     QUnit.push(
-      element.attributes.length === expectedCount,
+      element.attributes.length === expectedCount || !attributes,
```

Comment:
Not sure if this is kosher, but seemed strange to me that you couldn't use `assertElement` without explicitly expecting certain attributes. For some assertions, attributes are irrelevant (see `renders a contained view with omitted start tag and tagless parent view context`).